packages/k8s-triliovault-operator/k8s-triliovault-operator.patch

```diff
@@ -4,7 +4,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-origin
 @@ -10,3 +10,6 @@
  sources:
  - https://github.com/trilioData/k8s-triliovault-operator
- version: v2.0.2
+ version: v2.0.5
 +annotations:
 +  catalog.cattle.io/certified: partner
 +  catalog.cattle.io/release-name: k8s-triliovault-operator
```

Comment:
Could you please add this additional annotation for the name that will be displayed in the UI?
`catalog.cattle.io/display-name: TrilioVault Kubernetes Operator`
I used TrilioVault Kubernetes Operator as an example, but you can change it to whatever you think is best.

Reply:
Sure, I will fix this into the new commit